test/model/knock/auth_token_test.rb

```diff
@@ -47,5 +48,23 @@ class AuthTokenTest < ActiveSupport::TestCase
         AuthToken.new token: token
       }
     end
+
+    test "should validate exp claim when is set" do
```

Comment:
We can drop the "should" for consistency: "validate expiration claim by default"